src/server/index.js

```diff
@@ -68,10 +67,9 @@ app.get("/sw.js", (req, res) => {
   res.sendFile(path.resolve(__dirname, "build", "sw.js"));
 });
 
-/********************** Authentication **********************/
+/*************************** Authentication ***************************/
 
 app.use('/api/v1/', auth, function (req, res, next) {
```

Comment:
I noticed you remove the `auth` import in this file.

Please remove the `auth` param here.  if not it crashes the backend. 


![image](https://user-images.githubusercontent.com/15785697/65823501-73718e80-e225-11e9-8739-f7539b09a1dd.png)

Reply:
It was removed cos of the merge. Thanks for noticing!!